### ISSUE TITLE
Make the iframe adapt to content height dynamically

### ISSUE THREAD

**Issue by lakruzz** [Praqma/code-conf.com#25]
The ticket include uses an iframe, with  a fixed size, but dependent on how many tickets are available for an even, the hight may differ:

![image](https://cloud.githubusercontent.com/assets/155492/14505098/f88cd1f8-01b7-11e6-927f-d43a3e884e87.png)

![image](https://cloud.githubusercontent.com/assets/155492/14505113/0f33e0d6-01b8-11e6-8e5a-3c6840f79f43.png)

Make it adjust atomatically

**Comment by lakruzz** [comment]
http://stackoverflow.com/questions/9162933/make-iframe-height-dynamic-based-on-content-inside-jquery-javascript

**Comment by illus0r** [comment]
@lakruzz Struggled with this more than 3 hours, time to take a break.

Suggested approach and [another one](https://github.com/house9/jquery-iframe-auto-height) doesn't work because of cross-domain problem. I tried [a resizing lib for cross domain iframes](https://github.com/davidjbradshaw/iframe-resizer) but no way.

Can I spend 3 hours more on that? I'll set a task for freelancers

**Comment by lakruzz** [comment]
Let's put it on hold for a while

**Comment by illus0r** [comment]
Ok!

**Comment by illus0r** [comment]
@lakruzz I've sent this into backlog milestone. I had to create one.

**Comment by lakruzz** [comment]
@illus0r  anything not assigned to a specific milestone, is on the backlog :-)

**Comment by lakruzz** [comment]
Closed #108 as "won't have" we're going to do this one instead

**Comment by lakruzz** [comment]
Couldnt' make this work dynamically from http to https.

Change the height of the evenbrite include so it's default 300 - but be altered with a parameter like this:

```
{% include eventbrite-ticket-form.html
eventbriteid=page.checkout.eventbriteid
css_classes="black toprip botrip"
header="Enroll"
height="310"%}
```
